### PR TITLE
Create week3.md

### PR DESCRIPTION
**Pull request by penpenLoveUwU**

### CODE REVIEW

**Review by ApolloZhu — CHANGES_REQUESTED** [review]
1. There were 2 pull requests requiring review from week 3, see https://github.com/UWRealityLab/xrcapstone22wi-team8/issues/13
2. Let’s capitalize all list items (i.e. capitalize “b” in “brush” on line 20)
3. Course staff’s feedback for Week 4 snippet says that our summary is too vague, but I’m not sure how exactly we can be more detailed

File: docs/content/en/snippets/week3.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+There's nothing needing review this week.
+
+## Blocking Issues
+
+There are no blocking issues this week and we don't need other assistance.
+
+## What's Next
+
+Next week, we'll refine our individual functions and pull everything together and hopefully and the "multiuser".
```
Comment (ApolloZhu) [inline docs/content/en/snippets/week3.md:34]:
```suggestion
Next week, we'll refine our individual functions and pull everything together, and hopefully the "multiuser".
```

Comment (penpenLoveUwU) [reply docs/content/en/snippets/week3.md]: I have fixed this and the "Brush". I think it is fine to have week3 like this because this is before week4.

Comment (ApolloZhu) [reply docs/content/en/snippets/week3.md]: I see. Please add the two PRs for review by staff with one sentence description for each, then this should be good to go

**Review by penpenLoveUwU — COMMENTED** [review]
update again

**Review by penpenLoveUwU — COMMENTED** [review]
hi

**Review by ApolloZhu — APPROVED** [review]
Thanks for the updates

**Pull request merged**